app_dart/lib/src/request_handlers/update_discord_status.dart

```diff
@@ -31,7 +32,7 @@ final class UpdateDiscordStatus extends GetBuildStatus {
   Future<Body> get() async {
     // For now, limit these to flutter/flutter only
     if (request!.uri.queryParameters[_kRepoParam] != 'flutter') {
-      return Body.forJson(const <String, dynamic>{});
+      throw const BadRequestException('Only ?repo=flutter is supported');
     }
```

Comment:
I think we can default this to "flutter" since the super class defaults.

```suggestion
  if (request!.uri.queryParameters[_kRepoParam] case final repo?) {
    if (repo != 'flutter') {
        throw const BadRequestException('Only ?repo=flutter is supported');
    }
  }
```